src/RestControllers/InsuranceCompanyRestController.php

```diff
@@ -35,7 +35,7 @@ public function getAll()
 
     public function getOne($iid)
     {
-        $serviceResult = $this->insuranceCompanyService->getOne($iid);
+        $serviceResult = $this->insuranceCompanyService->getOneById($iid);
```

Comment:
the rest controllers should not be changed (the rest endpoints will use uuid then)
Just need to ensure all the other placed in codebase that use the service are changed

Reply:
just checked and otherwise looks good in codebase (just need to change this back to getOne() )
And then need to update lines 1479 and 1484 in API_README.md to show use of uuid.

Reply:
actually, that is a put. agree best thing to do here is to keep as is (changing to uuid for this restcontroller will require a bunch more more). so keep the getOneById call :)